src/screens/benefit/Details.tsx

```
@@ -117,20 +118,51 @@ const BenefitsDetails: React.FC = () => {
 			);
-		} else {
-			setLoading(true);
+			setLoading(false);
+			return;
+		}
+
+		// Step 3: Apply application
+		try {
 			const result = await applyApplication({ id, context });
+
 			const url = (result as { data: { responses: Array<any> } }).data
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)